using-timescaledb/backup.md

```diff
@@ -331,6 +328,26 @@ end as the recovery will be complete when no further files can be
 found in the archive. See the PostgreSQL documentation on
 [continuous archiving][pg archiving] for more information.
 
+### Named restore point [](named_restore_point)
+
+TimescaleDB natively support a consistent restore point both for single-node and for
+multi-node environments. The restore point can later be used as a recovery target on
+single-node or on each node of the entire multi-node cluster, ensuring that it can be
+restored to a consistent state.
+
+To create a named restore point on a single-node [`pg_create_restore_point`][pg_create_restore_point]
```

Comment:
What's the advantage of this in single-node?  

I think the above paragraph introduction should be a bit clearer about the user advantage of single vs. multi-node clusters.  In single this (I think) is more about ease-of-use --> just having a specific name to restore to.  In multi-node this is also (mainly?) about correctness/consistency.

Reply:
The advantage of using restore points in single node is the same as in multi-node, the idea here is that it expected to work in the same way. Maybe we should just concentrate on `distributed restore points` in such case and remove mentioning of the single-node?